### DISCUSSION TITLE
Account Suspension Review

### DISCUSSION THREAD

**Discussion by ToshiroSky** [community/community#140605] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hello GitHub Community,

I am reaching out because my GitHub account was recently suspended without any prior warning or explanation. I have been an active and contributing member of this community, and I am unsure what led to this action.

My account holds several important files and projects that I’ve worked on for a long time. Losing access to these would cause significant disruption to my work and progress. I deeply value the work I've done here and would like the opportunity to resolve any misunderstandings that may have occurred.

I kindly ask the community or any moderators to help guide me through the process of appealing this suspension or to provide insights into what might have triggered it. Any assistance would be greatly appreciated.

Thank you for your support and understanding.

Sincerely,
Suspensed account: CrimZonQH

**Comment by ebndev** [comment, marked as answer]
Hi @StPointer ,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
